src/components/common/notFounde/index.ts

```diff
@@ -0,0 +1 @@
+export {NotFound} from "./notFounde"
```

Comment:
опечатка в имени файла
```suggestion
export {NotFound} from "./notFound"
```